### DISCUSSION TITLE
Task list - show progress on pinned issue

### DISCUSSION THREAD

**Discussion by sussexrick** [community/community#4405] [Projects and Issues]
When an issue is pinned it has a card similar to the one on the boards in Projects. On the projects card the progress against the task list is shown, but not on the pinned issue - one of a number of differences. It would be useful to see the task list progress on the pinned issue too.

![image](https://user-images.githubusercontent.com/1260550/123848402-97d53000-d90f-11eb-9d5e-d3760e034ca2.png)

vs

![image](https://user-images.githubusercontent.com/1260550/123848434-a02d6b00-d90f-11eb-8d36-336f478dd187.png)

**Comment by weycen** [comment]
I also encountered the same problem. An issue containing multiple tasks can be used as a phased milestone. We hope to display the task progress of this issue at the top of the issue, but for now, the pinned issue cannot display task progress like normal issue.
